agent/tool-scripts/tests/cpuacct/gold/stderr

```diff
@@ -2,12 +2,20 @@ cpuacct: Error, one of the following options is required, --install|--start|--st
 
 The following options are available:
 
+Operational options - do *not* specify any of these when you register a tool:
+
 	--install                 verify this perf tool will work
 
 	--start|stop|postprocess  start/stop/postprocess the data collection
 	-d str|--dir=str          directory to store data collection (required)
+
+Tool-specific options - these should be specified when you register a tool:
+
 	--interval=int            number of seconds between each data collection (optional, default is 10 seconds)
 
```

Comment:
We should probably come up with a better word than "should".  (I.e., here at line 12 we're telling the user that `--interval` _should_ be supplied when, at line 14, we point out that it is **optional**).

Reply:
I changed it to "may".